core/App/components/record/RecordField.tsx

```diff
@@ -65,13 +66,18 @@ const RecordField: React.FC<RecordFieldProps> = ({
         </Text>
       )}
       <View style={styles.valueContainer}>
-        {fieldValue ? (
+        {fieldValue ? 
+        (
           fieldValue(field)
         ) : (
           <>
             <View style={styles.valueText}>
               <Text style={styles.text} testID={testIdWithKey('AttributeValue')}>
-                {shown ? (field as Attribute).value : Array(10).fill('\u2022').join('')}
+                {shown ? 
+                dateAttributes.includes(field.name!) ?
+                new Date((field).value * 1000).toLocaleDateString('en-CA', dateFormatOptions) :
```

Comment:
```suggestion
                new Date((field).value * 1000).toLocaleDateString('en-US', dateFormatOptions) :
```

I would suggest changing to the US format (unless I'm missing something here)

Reply:
Nope, you're totally right, my brain thought it stood for English California for some reason haha

Reply:
This has now been updated to be a constant imported from the constants file and switched to US.